File: gomax-angular/src/app/seance/models/seance.ts
import { Film } from 'src/app/film/models/film';
import { Salle } from 'src/app/salle/models/salle';


export class Seance {
    constructor(
        public id: number,
        public date_seance?: Date,
        public salle?: Salle,
        public film?: Film,

    ) { }
}

File: gomax-spring/src/main/java/com/gomax/entities/Seance.java
package com.gomax.entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import javax.persistence.*;
import java.time.LocalDate;
import java.time.LocalDateTime;


@Entity
@Table(name="Seance")

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Seance {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private LocalDateTime date_seance;

    @ManyToOne
    private Salle salle;
    @ManyToOne
    private Film film;


}